recipes/libcurand/meta.yaml

```diff
@@ -0,0 +1,154 @@
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 7c25712016fdde96011c3370673d9007eddd7231261a9b47f36b213ceb964191  # [linux64]
+  sha256: ac020f6e88105927f96f58ca331d3cddba35b37376f4716e866155c4f56681dd  # [ppc64le]
+  sha256: c7ae331dd6c9936a4e9746abdc222514bbda47a1e495e396b8244b918de377e2  # [aarch64]
+  sha256: 5fb95fbbd1c652e1e2a8c12c41d225117cab3f718c739f0d49d580c11b4a44f1  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+outputs:
+  - name: libcurand
+    files:
+      # Include SONAME and full name libraries, but not the linker name. That
+      # goes in the -dev package.
+      - lib/libcurand.so.*                            # [linux]
+      - targets/{{ target_name }}/lib/libcurand.so.*  # [linux]
+      - Library\bin\curand*.dll                       # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcurand.so.{{ version }}                                          # [linux]
+        - test -L $PREFIX/lib/libcurand.so.{{ version.split(".")[0] }}                            # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcurand.so.{{ version }}                # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libcurand.so.{{ version.split(".")[0] }}  # [linux]
+        - if not exist %LIBRARY_BIN%\curand64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      home: https://developer.nvidia.com/curand
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: cuRAND native runtime libraries
+      description: |
+        cuRAND native runtime libraries
+      doc_url: https://docs.nvidia.com/cuda/curand/
+
+  - name: libcurand-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcurand", max_pin="x") }}
+    files:
+      - lib/libcurand.so                            # [linux]
+      - lib/pkgconfig                               # [linux]
+      - targets/{{ target_name }}/include           # [linux]
+      - targets/{{ target_name }}/lib/libcurand.so  # [linux]
+      - targets/{{ target_name }}/lib/stubs         # [linux]
+      - targets/{{ target_name }}/libcurand/src     # [linux and not aarch64]
```

Comment:
The `src` directory exists for libcublas and libcusparse, which define Fortran bindings as C++ files in that directory. However, I don't see a `src` directory being shipped in libcurand for linux-x86_64 or linux-sbsa. Let's remove since it doesn't exist.
```suggestion
```

Reply:
Yeah there isn't an src currently, I had just left it in as future-proofing since I thought I'd seen discussions somewhere about wanting to minimize changes. I'm good with removing for now and adding later.

Reply:
We didn't do this for other math libraries without `src`, so we can just align.